packages/frontend/src/pages/Tiles/components/CreateTileButton.tsx

```diff
@@ -1,62 +1,199 @@
+import { BiSpreadsheet, BiTable } from 'react-icons/bi'
 import { HiOutlineHandRaised } from 'react-icons/hi2'
 import { useNavigate } from 'react-router-dom'
 import { useMutation } from '@apollo/client'
 import {
+  Flex,
   Modal,
   ModalBody,
   ModalContent,
   ModalFooter,
   ModalHeader,
   ModalOverlay,
+  Text,
   useDisclosure,
 } from '@chakra-ui/react'
 import {
   Button,
   FormLabel,
   Input,
   ModalCloseButton,
+  Tile,
 } from '@opengovsg/design-system-react'
 import { TILES_FEATURE_FLAG } from 'config/flags'
 import * as URLS from 'config/urls'
 import { LaunchDarklyContext } from 'contexts/LaunchDarkly'
 import { CREATE_TABLE } from 'graphql/mutations/create-table'
+import { ImportCsvModalContent } from 'pages/Tile/components/TableBanner/ImportCsvButton'
+import { TableContextProvider } from 'pages/Tile/contexts/TableContext'
 
-const CreateTileButton = (): JSX.Element => {
-  /**
-   * Check if feature flag is enabled, otherwise dont allow creation
-   * eventually, this should be removed
-   */
-  const { flags } = useContext(LaunchDarklyContext)
-  const isTilesEnabled = flags?.[TILES_FEATURE_FLAG]
+type TILE_CREATE_MODE = 'import' | 'new'
 
-  const { isOpen, onOpen, onClose } = useDisclosure()
-  const navigate = useNavigate()
+const CreateTileForm = ({
+  tableName,
+  setTableName,
+  onSubmit,
+  isSubmitting,
+  onImportNext,
+}: {
+  tableName: string
+  setTableName: (tableName: string) => void
+  onSubmit: () => Promise<void>
+  isSubmitting: boolean
+  onImportNext: () => void
+}) => {
+  const [createMode, setCreateMode] = useState<TILE_CREATE_MODE | null>(null)
 
+  return (
+    <>
+      <ModalHeader>Create a Tile</ModalHeader>
+      <ModalCloseButton />
+      <ModalBody>
+        <FormLabel isRequired>Enter a name for your new tile</FormLabel>
+        <Input
+          value={tableName}
+          onChange={(e) => setTableName(e.target.value)}
+        />
+        <FormLabel mt={8} isRequired>
+          How do you want to start?
+        </FormLabel>
+        <Flex gap={4}>
+          <Tile
+            icon={BiSpreadsheet}
+            flex={1}
+            onClick={() => setCreateMode('import')}
+            isSelected={createMode === 'import'}
+          >
+            <Text textStyle="h5" mt={4}>
+              Import CSV
+            </Text>
+            <Text textStyle="body-2">
+              Start with data that you have already collected
+            </Text>
+          </Tile>
+          <Tile
+            icon={BiTable}
+            flex={1}
+            onClick={() => setCreateMode('new')}
+            isSelected={createMode === 'new'}
+          >
+            <Text textStyle="h5" mt={4}>
+              Start from scratch
+            </Text>
+            <Text textStyle="body-2">Start with a blank database</Text>
+          </Tile>
+        </Flex>
+      </ModalBody>
+      <ModalFooter>
+        <Button
+          colorScheme="primary"
```

Comment:
super nit: is this necessary? I always thought if its not defined, it defaults to the primary color scheme